cli/src/main.rs

```diff
@@ -128,6 +133,26 @@ async fn main() {
                 .into_inner();
             info!("{:?}", connected_leaders);
         }
+        Commands::ConnectedLeadersInfo { rpc_url } => {
+            let connected_leaders_response = client
+                .get_connected_leaders(ConnectedLeadersRequest {})
+                .await
+                .expect("gets connected leaders")
+                .into_inner();
+
+            let rpc_client = RpcClient::new(rpc_url);
+            let epoch_info = rpc_client.get_epoch_info()
```

Comment:
what do you think about calling getVoteAccounts to get delegated stake + total stake? we've thought about trimming the response for get_connected_leaders to be only >= current slot instead of the entire epoch's. that would make this more reliable